### PR TITLE
Fix Travis CI pipeline issues

### PR DESCRIPTION
**Pull request by DTMad**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -124,35 +107,37 @@ jobs:
           - ws_amd
           - dockerconfig
 
-    - <<: *deploy
-      name: Deploy snapshot quay AMD
-      arch: amd64
-      if: (branch != master) AND (tag IS blank)
+    - <<: *deploy_amd
```
Comment (lrgar) [inline .travis.yml:110]: We're no longer publishing to Quay for releases, only development builds.

**Comment by lrgar** [comment]
Please also add an entry to changelog.

File: .travis.yml (hunk 1 above)
Comment (DTMad) [reply .travis.yml:110]: Changed condition to "if: (branch != master) AND (tag IS blank)"

Comment (lrgar) [reply .travis.yml:110]: Suggest renaming it to "Deploy snapshot quay AMD"

Please also do the name/if changes to the ARM job.

Comment (DTMad) [reply .travis.yml:110]: Thanks for the hint! Done

**Pull request merged**